### DISCUSSION TITLE
GPU support?

### DISCUSSION THREAD

**Discussion by amutz** [community/community#28490] [Codespaces]
I use github codespaces and I love it.  I would like to also use it to develop ML models, and for this to be efficient, I need to use a GPU.

From reading the documentation, it looks like codespaces does not have an option for a VM that has a GPU.

I am considering switching to coder.com in order to develop on cloud VM instances that I manage, in order to have access to a GPU, but wanted to check if they are coming soon?

Please provide any information you can on when we can expect to see codespaces offer a GPU VM option.

Thank you.

**Comment by jshorty** [comment]
Hi there! If you don't mind reaching out to [codespaces@github.com](mailto:codespaces@github.com) with this request, we can provide some info, thank you!
